### PR TITLE
Toggleable dark background for post titles

### PR DESCRIPTION
**Pull request by johnmccabe**
This commit allows the following to be set in the YAML Front Matter which will turn on span background colouring for a post title and description.

    darkbackground: true

screen shot 2018-08-22 at 14 10 13

### CODE REVIEW

**Comment by johnmccabe** [comment]
This fixes #18

**Comment by johnmccabe** [comment]
derek assign: alexellis

**Comment by johnmccabe** [comment]
@alexellis

File: _posts/2018-08-01-gke-multi-stage.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ categories:
   - kubernetes
   - gke
 author_staff_member: stefan
+darkbackground: true
```
Comment (alexellis) [inline _posts/2018-08-01-gke-multi-stage.md]: I'm nit picking, but can it be `dark_background`? Also Jekyll is awesome and so are you 😄

**Comment by alexellis** [comment]
How does it look on mobile?

File: _posts/2018-08-01-gke-multi-stage.md (hunk 1 above)
Comment (johnmccabe) [reply _posts/2018-08-01-gke-multi-stage.md]: done

**Comment by johnmccabe** [comment]
Renders fine on mobile, overflow is a different issue however and won't be dealing with it here (new PR incoming for that).

**Comment by alexellis** [comment]
Really nice, clean PR for this. It will be a good example to show other people how to extend the blog theme.

**Pull request merged**